spynnaker/pyNN/models/pynn_population_common.py

```diff
@@ -201,17 +207,22 @@ def __getitem__(self, index_or_slice):
         # TODO: Used to get a single cell - not yet supported
         raise NotImplementedError
```

Comment:
(It looks like there are a few more further down the file as well)

Reply:
This is a bit weird but I think Christian will resolve this in his latest changes.  Basically, this *is* implemented, but only in sPyNNaker8.  Since we only *have* sPyNNaker8 now, this is implemented!

For the later on one, the error is the error is "never to be implemented ever" so this works differently.